<launch>
	<group ns="action_modulation">
		<param name="platform" value="triskar_small" />
		<include file="$(find theatrebot_action_modulation)/action_modulation_nodes.xml" />
	</group>

<node pkg="rosserial_python" type="serial_node.py" name="serial_node">
	<param name="port" value="/dev/ttyACM0" />
	<param name="baud" value="115200" />
</node> 
</launch>
